pkgs/tools/security/grap/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchFromGitHub, pkgs, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "grap";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "QuoSecGmbH";
+    repo = "grap";
+    rev = "v${version}";
+    sha256 = "1fkdi7adfffxg1k4h6r9i69i3wi93s44c1j4cvr69blxsfh0mcnc";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    boost.all
+    libseccomp
+    flex
+    python3Packages.pefile
+    python3Packages.pyelftools
+    python3Packages.capstone
+    swig4
+    bison
+    cmake
+  ];
+
+  configurePhase = ''
+    ls $src
+    mkdir build;
+    cd build;
+    cmake -DCMAKE_INSTALL_PREFIX=$prefix -DPYTHON_SITE_DIR=$out/${python3.sitePackages} ../src
+  '';
```

Comment:
```suggestion
  cmakeFlags = [
    "-DPYTHON_SITE_DIR=$out/${python3.sitePackages}"
  ];
```

Reply:
How do I configure it to use the 'src' directory as the root of the cmake?

Reply:
nm. Done

Reply:
I think you are searching for sourceroot.